bouncer.go

```diff
@@ -151,11 +152,18 @@ func New(ctx context.Context, next http.Handler, config *configuration.Config, n
 				return nil, err
 			}
 		}
+		if err := handleStreamCache(bouncer); err != nil {
+			return nil, err
+		}
+		isStartup = false
 		ticker = startTicker(config, func() {
-			handleStreamCache(bouncer)
+			if err := handleStreamCache(bouncer); err != nil {
+				isCrowdsecStreamHealthy = false
```

Comment:
Here, the log has been moved from the inner function to the caller, which is good because errors can be handled this way better.

But I assume it will generate as much more logs than before.

Reply:
I would suggest that we add a parameter that register the number of time that the stream has been `unhealthy`

This will be a parameter of a function calculating the refresh period.
Let say period = (initial_period + 60 sec * nb_failed_healthy)

Once stream is healthy this parameter would be resetted to 0.

This would reduce the number of logs and the pressure until the stream is healthy again